packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -644,7 +645,13 @@ abstract class TestRunner {
         );
       });
     } else if (watchEntity.doc) {
-      const d = doc(watchEntity.doc[0], watchEntity.doc[1], watchEntity.doc[2]);
+      let d;
+      if (watchEntity.doc[2]) {
```

Comment:
This code is unreadable. :-)  Can you add:

```
const [key, version, data] = watchEntity.doc;
```
and use the named variables?

Reply:
I was going to use this part for my Go readability, but I guess not then.